database/v1.0.sql

```diff
@@ -8,7 +8,7 @@ CREATE TABLE IF NOT EXISTS `cb_contactus_messages` (
   `address_number`          varchar(10) DEFAULT NULL,
   `address_neighborhood`    varchar(80) DEFAULT NULL,
   `address_complement`      varchar(500) DEFAULT NULL,
-  `address_cep`             int(8) unsigned DEFAULT NULL,
+  `address_cep`             varchar(8) DEFAULT NULL,
```

Comment:
Qual o motivo que o levou a mudar o tipo do campo?

Reply:
A intenção era essa desde o início, na classe está como string. Eu vi em algum lugar, não lembro onde, que telefones são melhores de serem armazenados como string, também não lembro o motivo. Por isso achei que seria melhor representar o CEP com uma string.

Reply:
Nesse link aqui tem alguns pontos favoráveis ao armazenamento em varchar em vez de int @johnnysaymon 
https://alexandreaquiles.com.br/2014/02/18/porque-armazenar-numericos-como-string/

Reply:
Ótimo. Concordo.